src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,66 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String DATA_FIELD_SEPARATOR = ",";
+    private static final int OPERATION_TYPE_INDEX = 0;
+    private static final int AMOUNT_INDEX = 1;
+    private static final String SUPPLY_FIELD_NAME = "supply";
+    private static final String BUY_FIELD_NAME = "buy";
+    private static final String RESULT_FIELD_NAME = "result";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        writeToFile(toFileName, processData(readDataFromFile(fromFileName)));
```

Comment:
If you split these methods into three separate lines, it would improve the readability of your code. 

> String[] dataFromFile = readDataFromFile(fromFileName);
> String processedData = processData(dataFromFile);
> writeToFile(toFileName, processedData)

However, the current implementation is also acceptable. In general, calling all methods in a single line is not considered a best practice.